doc/lua_api.md

```diff
@@ -8057,7 +8057,9 @@ A 32-bit pseudorandom number generator.
 Uses PCG32, an algorithm of the permuted congruential generator family,
 offering very strong randomness.
 
-It can be created via `PcgRandom(seed)` or `PcgRandom(seed, sequence)`.
+* constructor `PcgRandom(seed, [seq])`
+  * `seed`: 64-bit unsigned seed
```

Comment:
Not this PR's job to fix, but this is fishy since on the Lua side of things, we can only represent integers up to 2^53 accurately.

Reply:
:shrug: